lib/scaleway/bootstrap/ks-master-cluster.tf

```diff
@@ -0,0 +1,22 @@
+resource "scaleway_k8s_cluster" "kubernetes_cluster" {
+  name    = "test-cluster" # TODO(benjaminch) : use : "qovery-${var.kubernetes_cluster_id}"
+  version = "1.20"
+  cni     = "cilium"
+
+  tags    =  [for i, v in local.tags_ks : "${i}=${v}"] # NOTE: Scaleway doesn't support KV style tags
```

Comment:
You should declare a local variable and set it directly there. Then call this variable on every tags